src/main/scala/tilelink/Bundles.scala

```diff
@@ -44,6 +44,41 @@ object TLMessages
 
   def adResponse = Vec(AccessAck, AccessAck, AccessAckData, AccessAckData, AccessAckData, HintAck, Grant, Grant)
   def bcResponse = Vec(AccessAck, AccessAck, AccessAckData, AccessAckData, AccessAckData, HintAck, ProbeAck, ProbeAck)
+  
+  def a = Seq( ("PutFullData",TLPermissions.PermMsgReserved),
+               ("PutPartialData",TLPermissions.PermMsgReserved),
+               ("ArithmeticData",TLAtomics.ArithMsg),
+               ("LogicalData",TLAtomics.LogicMsg),
+               ("Get",TLPermissions.PermMsgReserved),
+               ("Hint",TLHints.HintsMsg),
+               ("AcquireBlock",TLPermissions.PermMsgGrow),
+               ("AcquirePerm",TLPermissions.PermMsgReserved))
```

Comment:
Oh. This should be PermMsgGrow.

Reply:
Well they are for the reserved fields. So i put them all in 1 object and use the same seq in other places. Do you want me to change is accordingly?

Reply:
OOh sorry my bad. I'll cross verify it

Reply:
The AcquirePerm Is not there in the TileLink Spec. Draft 1.7